app/models/truck.rb

```diff
@@ -0,0 +1,7 @@
+class Truck < ApplicationRecord
+  validates :license_plate, presence: true
+  validates :capacity, presence: true
+  enum status: [ :active, :inactive, :servicing ]
+
+  validates_inclusion_of :status, in: Truck.statuses.keys
```

Comment:
we need a test for this.
also please use
`validates :status, inclusion: { in: Truck.statuses.keys }`
We dont use the fullname validation helpers like `validates_inclusion_of`

Reply:
In case we have `enum status...` validation won't work as it will raise an exception on initialization, and won't get to the validation stage.

Reply:
are you sure? whats a case where it raises an exception?